.travis.Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM dsluiuc/honeybadgermpc-docker-base
```

Comment:
okay so this is essentially only for travis to use right

Reply:
This change is not development friendly, if I modify the Dockerfile in `honeybadgermpc` then it requires me to modify this file as well (which I might not even know that I have to do).

We had earlier made an effort to unify our Dockerfiles (one file for both Travis and dev). Can we keep it that way?

Reply:
Let's address this in the next PR.